File: src/Lista01/Questao01_02/dados/Cliente.java
package Lista01.Questao01_02.dados;

public class Cliente {
    private String nome;
    private String cpf;
    private Endereco endereco;
    private Veiculo veiculoAlugado;

    public Cliente() {
    }

    public Cliente(String nome, String cpf, Endereco endereco, Veiculo veiculoAlugado) {
        this.nome = nome;
        this.cpf = cpf;
        this.endereco = endereco;
        this.veiculoAlugado = veiculoAlugado;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public void setVeiculoAlugado(Veiculo veiculoAlugado) {
        this.veiculoAlugado = veiculoAlugado;
    }

    public void setEndereco(Endereco endereco) {
        this.endereco = endereco;
    }

    public String toString() {
        return "Nome: " + this.nome + ", Cpf: " + this.cpf + ", Endereço: " + this.endereco.toString()/* + ", Veículo alugado: " + veiculoAlugado.toString()*/;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Cliente cliente = (Cliente) o;
        return nome.equals(cliente.nome) && cpf.equals(cliente.cpf) && endereco.equals(cliente.endereco) && veiculoAlugado.equals(cliente.veiculoAlugado);
    }

}

File: src/Lista01/Questao05/Cliente.py
import string
from Endereco import Endereco
from Veiculo import Veiculo

class Cliente:

	def __init__(self, nome, cpf, endereco, veiculo):
		self.__nome = nome
		self.__cpf = cpf
		self.__endereco = endereco
		self.__veiculo = veiculo

	def getNome(self):
		return self.__nome
	
	def setNome(self, nome):
		self.__nome = nome
	
	def getCpf(self):
		return self.__cpf
	
	def setCpf(self, cpf):
		self.__cpf = cpf
	
	def getEndereco(self):
		return self.__endereco
	
	def setEndereco(self, endereco):
		self.__endereco = endereco
	
	def getVeiculo(self):
		return self.__veiculo
	
	def setVeiculo(self, veiculo):
		self.__veiculo = veiculo
	
	def __str__(self):
		return "Nome: {}, Cpf: {}, Endereço: {}, Veículo Alugado: {}".format(self.__nome, self.__cpf, self.__endereco, self.__veiculo)